script/test/cmd/tests.sh

```diff
@@ -43,5 +43,13 @@ convert::expect_success_and_warning "kompose convert --stdout -f $KOMPOSE_ROOT/s
 convert::expect_success_and_warning "kompose convert --stdout --dc -f $KOMPOSE_ROOT/script/test/fixtures/entrypoint-command/docker-compose.yml" "$KOMPOSE_ROOT/script/test/fixtures/entrypoint-command/output-os.json" "Service cannot be created because of missing port."
 
 
+######
+# Tests related to docker-compose file in /script/test/fixtures/volumes_from
+# kubernetes test
+convert::expect_success_and_warning "kompose convert --stdout -f $KOMPOSE_ROOT/script/test/fixtures/volumes_from/docker-compose.yml" "$KOMPOSE_ROOT/script/test/fixtures/volumes_from/output-k8s.json" "ignoring path on the host"
```

Comment:
I am facing problems with this test, it fails sometimes, and I am trying to debug it!